packages/titanium-address-input/src/manual-address-dialog.ts

```diff
@@ -19,8 +19,10 @@ export class ManualAddressDialogElement extends LitElement {
   @property({ type: String }) label: string = '';
   @property({ type: String }) street: string = '';
   @property({ type: String }) city: string = '';
+  @property({ type: String }) county: string = '';
   @property({ type: String }) state: string = '';
   @property({ type: String }) zip: string = '';
+  @property({ type: Boolean }) showCounty: boolean;
```

Comment:
```suggestion
  @property({ type: Boolean, attribute: 'show-county' } }) showCounty: boolean;
```

Can we normalize the attribute for this?